### ISSUE TITLE
gg=G auto indent multiline formatting

### ISSUE THREAD

**Issue by Eclipse-Dominator** [nus-cs2030/2122-s1#93]
I split Optional.ifPresentOrElse into multiple lines to make it look more readable. But both my local vim autoformat and the one on pe node will format it to the incorrect indentation level required by checkstyle. Does any VIM expert here know any quick way to resolve this?

**Comment by RussellDash332** [comment]
Somehow, gg=G and the regular Enter autoindent shifts the spacing by 8 spaces while checkstyle wants it to be 4 spaces further. Not sure why but I anyhow did the manual work to get things done.

**Comment by huzaifa1712** [comment]
Yeah I had to do it manually as well to get it to pass checkstyle, wonder if anyone has a better solution.

**Comment by hwkchia** [comment]
Hmm... don't know if this will work... in vim `:set cino+=(4`.

**Comment by Eclipse-Dominator** [comment]
hmm so cinoptions+=(4 indents 4 characters from the line with unclosed parenthesis which solves the indentation until the closing bracket. 
After reading the documentation for VIM indent, you need to add 
`:set cino+=(4, m1` where `(4` indent 4 char for subsequent lines with unclosed parenthesis and `m1` aligns the closing parenthesis with the first character of the line with the matching opening parenthesis
